docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1327,6 +1327,27 @@ The ACS will be used for V&V testing. It is defined as:
 - **Not** longitudinal (independent samples) 
 - Includes mail/online, telephone, and personal visits 
 
+**Survey - Current Population Survey (CPS)** 
+
+The CPS is a survey run by the Census Bureau and gathers data about the 
+labor force, employment and unemployment, demographics, earnings, and 
+more information. It is an important survey and therefore is being 
+added here. 
+
+To create this survey: 
+
+- Sample rate of (60,000 / 350 million)% 
+- Sample will be stratified by state to ensure an even distribution 
+- Sample at each time step (approximates monthly) for 12 time steps 
+- **Not** longitudinal (independent samples) 
+- This survey utilizes personal visits and phone calls. As this does not fit into the framework above, we will use the values for mail/online, telephone, and personal visits and then apply an overall non-response rate of 27.6%. This additional risk of non-response will be added to all simulants regardless of race/ethnicity, age, or sex 
```

Comment:
There has been discussion about having the simulation's observers output the "true" state of the world and then having processes that operate outside the simulation itself use that data to generate realistic census data, household surveys, etc. enabling all the noising and sampling rates to be configurable. If that is the approach we are moving in, would we want response rate and/or sample rate to be configurable in that step? If so, we'd need to record more data than we'd expect the survey to actually record.

Reply:
This is a good question and @aflaxman would like your thoughts. 

My plan was to have response rate/sample rate in the observer and then primarily add noise afterwards. Noise would be things like misspelling names/addresses, etc. 

But I think we in general need to better define what should happen in the simulation, in the observer and outside of both. For example duplicating people - is that in the simulation or the observer or outside of it?

Reply:
Agree. Some things will need to be in the simulation, and some things will be easier if they are in the simulation.  Everything else (e.g. OCR errors), let's do as post-processing after the sim has been run.